Automation for creating a sqlite package for redistributing sqlite3.dll

natemcmaster opened the pull request:
Here is a KoreBuild friendly setup that builds a sqlite package with native windows components only.

cc @divega @moozzyk @eilon @davidfowl @bricelam

Commit:
Prototype building a sqlite package for redistributing sqlite3.dll

natemcmaster commented:
cc @mistachkin

moozzyk commented on tools/PackageBuilder/Program.cs:
```diff
@@ -0,0 +1,43 @@
+                    Directory.SetCurrentDirectory("src/libsqlite3.native");
+                    NativePackage.Build(SqliteVersion);
+                }
+                finally
```
nit: could you avoid all nested try/catch and have one finally after the exception handler below?

moozzyk commented:
I think you might be missing some legal stuff (e.g. we have [this](https://github.com/aspnet/libuv-package/blob/dev/content/thirdpartynotices.txt)) in libuv. @Eilon will probably know what's needed.

Other than that LGTM

Commit:
Remove korebuild

natemcmaster commented:
Updated. I added the "public domain" to the copyright. Also, it looks like we won't be running this on our own build server. I've stripped out the korebuild part and updated to the package names.

bricelam commented:
LGTM
